app/helpers/application_helper.rb

```diff
@@ -887,10 +887,18 @@ def custom_css_url
     end
   end
 
-  # all courses now use bz_newui.css, so we just rename it here.
-  # the name change is kept for now for cross-repo compatibility
+  # all pages that aren't `new_html` now use bz_newui.css.
+  # the "bz_newui" name change is kept for now for cross-repo compatibility
   def bz_css_choice(orig)
-    orig.sub("bz_custom.css", "bz_newui.css")
+    if defined? @page and defined? @page.new_html? and @page.new_html?
```

Comment:
this is hard to read, but I tried a few different ways and didn't like any of them :/